build/scripts/sdk-update-version/main.go

```diff
@@ -53,6 +53,7 @@ func main() {
 		"sdks/unity/package.json",
 		"sdks/csharp/sdk/AgonesSDK.nuspec",
 		"sdks/csharp/sdk/csharp-sdk.csproj",
+		"sdks/rust/Cargo.toml",
```

Comment:
Just confirming - looks like this does the Cargo rust updates in this one place, hence deleting the target and removal from the release checklist?

Reply:
yes, that's correct🙂

Reply:
Nice!